scripts/dpf-sanity-checks.sh

```
@@ -0,0 +1,453 @@
+dpu_workers=()
+
+dpu_worker_count=0
+
+for i in $dpu_worker_list ; do 
+  dpu_workers[${dpu_worker_count}]=$i
+  echo "Worker node '${dpu_workers[${dpu_worker_count}]}' is Ready";
+  ((dpu_worker_count++))
+done
+
+echo "Detected ${dpu_worker_count} DPU workers in Ready phase"
+echo -e "Working with ${dpu_worker_count} DPU worker nodes: '${dpu_workers[@]}'"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)